src/lighteval/tasks/registry.py

```diff
@@ -221,7 +255,13 @@ def taskinfo_selector(
     if "." in tasks and os.path.exists(tasks):
         tasks = ",".join([line for line in open(tasks, "r").read().splitlines() if not line.startswith("#")])
 
```

Comment:
Since you're editing this section, could be nice to also simplify here - makes no sense to `",".join` here to then `.split(",")` below.
So either create a list directly if tasks is an openable file or split if tasks is a string of tasks